### ISSUE TITLE
VSCode-Test appears to be failing on Travis CI linux tests

### ISSUE THREAD

**Issue by glennsarti** [Microsoft/vscode-test#3]
From https://github.com/Microsoft/vscode/issues/70857

---

Okay...so some initial digging.

1. The `https://github.com/Microsoft/vscode-extension-vscode` repo which holds that npm module has changed it's tagging system from `1.1.30` to `v.1.1.32` which may be coincidental.

The release dates for [vscode on npmjs](https://www.npmjs.com/package/vscode) shows that 1.1.31 and 1.1.32 shortly thereafter were only a day ago so that explains why things have suddenly gone bad.  1.1.30 was a month ago.

A quick look at the diff between the two....
https://github.com/Microsoft/vscode-extension-vscode/compare/1.1.30...v1.1.32

Shows some major work on the `bin/test` file shows that the functionality seems to have moved to a `vscode-test` module (https://www.npmjs.com/package/vscode-test) which is VERY new (7 days ago)

Interestingly is this line: https://github.com/Microsoft/vscode-extension-vscode/compare/1.1.30...v1.1.32#diff-f900621cc6a1719b5473baf0191f297bR12

It says that the testing should be in the `stable` folder which clearly doesn't exist in my example above.  It did exist on my Ubuntu 18.04 desktop though

Given the following log from a job
```
Downloading VS Code into "/home/travis/build/lingua-pupuli/puppet-vscode/.vscode-test/stable
Downloading VS Code 1.32.3
Downloaded VS Code 1.32.3
```

My hunch is that `unzipVSCode` (https://github.com/Microsoft/vscode-test/blob/master/lib/download.ts#L127) is failing silently.'

Okay so there is no error trapping in the unzipVSCode method

https://github.com/Microsoft/vscode-test/blob/master/lib/download.ts#L88-L99

Given this is using `spawnSync` it may be worth trapping the exitcode, stdout and stderr and throw errors if the exit code is not zero.

Also may be worth raise an error the extractDir doesn't exist after an extract.

**Comment by octref** [comment]
This should be fixed on either `vscode@1.33` or `vscode-test`. Thanks for reporting!

**Comment by glennsarti** [comment]
@octref Confirmed 1.1.33 fixed my pipelines.  Thankyou for the quick turnaround.
